walkthroughs/howto-tls-file-provided/mesh/mesh.yaml

```diff
@@ -0,0 +1,157 @@
+Description: >
+  This template deploys the mesh and all of its components.
+
+Parameters:
+
+  MeshName:
+    Description: The name of the mesh to create
+    Type: String
+
+  ServicesDomain:
+    Description: The DNS suffice applied to virtual service names (e.g. default.svc.cluster.local)
+    Type: String
+
+  ColorTellerGreenRouteWeight:
+    Description: The weight to apply to the Green Color Teller Route
+    Type: Number
+    AllowedValues: [0, 1]
+    Default: 0
+
+  ColorGatewayTlsValidationPath:
+    Description: The path to use for TLS client validation on the Color Gateway
+    Type: String
+    Default: "null"
```

Comment:
So when `Enforce` is set to `False` we do not vend the configuration to the envoy. This allows us to put whatever we want in `CertificateChain` and the virtual node will not see it. If `Enforce` is `True`, then if there is no certificate chain in the provided location then the envoy will fail to start up. 

I'll switch this to use a default value of `AWS::NoValue` to avoid confusion though.